options/locale/locale_en-US.ini

```diff
@@ -1395,6 +1395,16 @@ editor.user_no_push_to_branch = User cannot push to branch
 editor.require_signed_commit = Branch requires a signed commit
 editor.cherry_pick = Cherry-pick %s onto:
 editor.revert = Revert %s onto:
+editor.fork_create = Fork Repository to Propose Changes
+editor.fork_create_description = You can not edit this repository directly. Instead you can create a fork, make edits and create a pull request.
+editor.fork_edit_description = You can not edit this repository directly. The changes will be written to your fork <b>%s</b>, so you can create a pull request.
+editor.fork_failed_to_push_branch = Failed to push branch %s to your repoitory.
+editor.cannot_find_editable_repo = Can not find repository to apply the edit to. Was it deleted while editing?
+editor.fork_not_editable = Fork Repository Not Editable
+editor.fork_internal_error = Internal error loading repository information about <b>%s</b>.
```

Comment:
I think we need to remove this translation, because all "internal errors" are reported as 500 page.

Here it's better to avoid re-inventing the wheel. Will make some changes later.

Reply:
I fixed this one as well, not sure it matches what you had in mind.